### ISSUE TITLE
Optimizer Learning Scheduler 

### ISSUE THREAD

**Issue by khurramsiddiqui** [optuna/optuna-examples#30]
## Motivation

lr_scheduler.ExponentialLR, lr_scheduler.ReduceLROnPlateau etc with respect to each optimizer Adam , RMSprop etc

## Description

As we have search space for finding different optimizer ADAM, RMSprop. Similarly can we have search space within Optuna for learning scheduler i.e. lr_scheduler.ExponentialLR, lr_scheduler.ReduceLROnPlateau etc with respect to each optimizer

**Comment by nzw0301** [comment]
Could you clarify your issue? Is it a feature request to add such an example to an existing DNN framework example code?

**Comment by khurramsiddiqui** [comment]
yes a feature request please , not sure whether these params lr_scheduler.ExponentialLR, lr_scheduler.ReduceLROnPlateau are already in search space of optuna or not?

**Comment by nzw0301** [comment]
I think no. But I think we can sample them via `suggest_categorical` like optimiser. In my machine learning knowledge, linear warmup and cosine annealing scheduler is a common way rather than the others. Anyway, if you are interested in such a feature, please send a PR!
